internal/model/metadata.go

```diff
@@ -6,7 +6,7 @@ type Metadata struct {
 	Bucket       string    `json:"bucket" db:"bucket"`
 	Name         string    `json:"name" db:"name"`
 	Parent       string    `json:"parent" db:"parent"`
-	StorageClass string    `json:"storage_class" db:"storage_class"`
+	StorageClass string    `json:"storageClass" db:"storage_class"`
```

Comment:
why make this change?

Reply:
Initially I tried to map directly into Metadata until I found out unmarshal doesn't cast to struct types (we can't).

I left it because it aligns with [Google JSON Style guide](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format) and it is the same key name used in pub sub messages